### ISSUE TITLE
Accessibility enhancement

### ISSUE THREAD

**Issue by danlong-stq** [timothyjensen/gravityformsadvancedselect#5]
Hi,

Thanks for the plugin, it's working really well! Are you able to add an aria-label (with a value) to the <select> menus being generated?

A website that this is used on needs to pass the WCAG 2.1 standards, and the <select> menu from this plugin is missing it's aria-label.

It would be great if it's something you're able to add on!

**Comment by timothyjensen** [comment]
@danlong-stq Thanks for raising this issue. From what I see the dropdown is properly labeled, however the native `<select>` element loses its label when the enhanced dropdown (TomSelect) is initialized. I addressed this in https://github.com/timothyjensen/gravityformsadvancedselect/commit/5a87dcb48510ba8af6d55433c324c06bea71c3d8 by adding `aria-hidden="true"` to the hidden select element when TomSelect initializes. Please let me know if this addresses your concern.

**Comment by danlong-stq** [comment]
That's perfect @timothyjensen. Thanks for the enhancement and additional info on what's happening under the hood!
